config/v1/feature_gates.go

```diff
@@ -206,6 +206,13 @@ var (
 				enableIn(TechPreviewNoUpgrade).
 				mustRegister()
 
+	FeatureGateSetEIPForNLBIngressController = newFeatureGate("SetEIPForNLBIngressController").
+							reportProblemsToJiraComponent("Routing").
+							contactPerson("miheer").
+							productScope(ocpSpecific).
+							enableIn(TechPreviewNoUpgrade).
```

Comment:
There's now also a DevPreview you probably want to enable it on.  `DevPreviewNoUpgrade, TechPreviewNoUpgrade`